bin/_docker.sh

```diff
@@ -52,6 +53,22 @@ docker_build() {
     echo "$repo:$tag"
 }
 
+docker_build_go() {
```

Comment:
this all seems unrelated to this change?

Reply:
> this all seems unrelated to this change?

Yes, bad branch management on my part. I removed that commit.